generics-sop/src/Generics/SOP/GGP.hs

```diff
@@ -137,15 +137,18 @@ instance GProductTo U1 where
 
 -- This can most certainly be simplified
 class GSumFrom (a :: Type -> Type) where
-  gSumFrom :: a x -> SOP I xss -> SOP I (ToSumCode a xss)
+  gSumFrom :: a x -> Proxy xss -> SOP I (ToSumCode a xss)
```

Comment:
This should probably be `proxy xss` (instead of `Proxy xss`) for symmetry with `gSumSkip`.